netbox/dcim/choices.py

```diff
@@ -1235,6 +1235,7 @@ class InterfaceTypeChoices(ChoiceSet):
                 (TYPE_100GE_CR2, '100GBASE-CR2 (100GE DAC)'),
                 (TYPE_100GE_CR4, '100GBASE-CR4 (100GE DAC)'),
                 (TYPE_100GE_CR10, '100GBASE-CR10 (100GE DAC)'),
+                (TYPE_100GE_CWDM4, '100GBASE-CWDM4 (100GE)'),
```

Comment:
We cannot accept the introduction of new types under the scope of a bug report. Please open a separate FR to propose its inclusion.

Reply:
Ah, that's my mistake. It appears that this was omitted by accident.